Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp

```diff
@@ -160,7 +160,7 @@ void CompositingCoordinator::sizeDidChange(const IntSize& newSize)
 
 bool CompositingCoordinator::flushPendingLayerChanges(OptionSet<FinalizeRenderingUpdateFlags> flags)
 {
-    TraceScope traceScope(BackingStoreFlushStart, BackingStoreFlushEnd);
+    TraceScope traceScope(FlushPendingLayerChangesStart, FlushPendingLayerChangesEnd);
```

Comment:
This changed a platform-agnostic `TracePointCode` to a GTK/WPE-specific one, so it also needs a new `#if` guard; will add.

Reply:
Aw, sorry @rkirsling - thanks for taking care!